lib/modules/commentTools.js

```diff
@@ -259,6 +259,12 @@ module.go = () => {
 		});
 	}
 
+	// slideEditBar on comments only
+	$body.on('input', 'form[id^=\'commentreply\'] textarea', function() {
```

Comment:
`updateCounter` listens for `input` because it needs to update the count on every input. That's not the case here. Right now this reattaches the listeners on every single keypress.

It looks like reddit does clone the textarea when you click "reply"--leaving the listeners attached through `attachEditorToUsertext` behind--so we should move the focus{in,out} listeners up here as delegated listeners (and get rid of this one).